systers_portal/meetup/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from users import SystersUser
+from cities_light.models import City
+
+
+class MeetUpLocation(models.Model):
+    """Manage details of MeetUp Location groups"""
+    name = models.CharField(max_length=255, unique=True, verbose_name="Name")
+    slug = models.SlugField(max_length=150, unique=True, verbose_name="Slug")
+    city = models.ForeignKey(City)
+    description = models.TextField()
```

Comment:
Please add a string representation to `MeetupLocation` and a test that will check if the method works ok.